go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/davidmparrott/kafka_exporter/v2 v2.0.1
 	github.com/drone/envsubst v1.0.2
 	github.com/fatih/structs v1.1.0
-	github.com/go-kit/kit v0.10.0
+	github.com/go-kit/kit v0.11.0
```

Comment:
There are a few drive-by updates in go.mod and go.sum. I'll confess to still being a bit of a n00b WRT go modules, so I *think* these are okay, but I am not 100% sure.

The agent still builds. and tests still pass, so I think we're good.

Part of the extensive nature of the change here is that the postgres exporter repo changed from `github.com/wrouesnel/postgres_exporter` to `github.com/prometheus-community/postgres_exporter`.

Due to my go mod inexperience, I did a `go mod tidy` which... May be wrong.